src/scenes/UserGalleryPage/LinkButton.tsx

```diff
@@ -3,12 +3,9 @@ import styled from 'styled-components';
 
 import CopyToClipboard from 'components/CopyToClipboard/CopyToClipboard';
 
-export default function LinkButton({ username }: { username: string }) {
+export default function LinkButton({ textToCopy }: { textToCopy: string }) {
   return (
-    <CopyToClipboard
-      textToCopy={`https://gallery.so/${username}`}
-      successText={'Copied link to clipboard'}
-    >
```

Comment:
Now, `LinkButton` takes in the `textToCopy` as a string so that it can be used for different URLs (galleries, collections, etc.)